src/CbcSolverInterface.jl

```diff
@@ -1,9 +1,27 @@
 module CbcMathProgSolverInterface
 
 using Cbc.CbcCInterface
+using Compat.SparseArrays
 
-importall MathProgBase.SolverInterface
+import MathProgBase
+const MPB = MathProgBase.SolverInterface
```

Comment:
Using `MPB` as an alias for anything except `MathProgBase` is confusing. Just use `const MPB = MathProgBase`. See Ipopt.jl for an example.

Reply:
I thought I followed some other package but must have been confused by the `importall`.